src/steps/assessment_projects/index.ts

```diff
@@ -35,28 +36,45 @@ export async function fetchAssessments({
       nextApplicationUri,
     );
     nextApplicationUri = nextUri;
-    const assessmentEntities = items.map((application) =>
-      createAssessmentEntity(application),
-    );
-    await jobState.addEntities(assessmentEntities);
-    const relationships = assessmentEntities.map((assessmentEntity) => {
-      return createDirectRelationship({
-        _class: RelationshipClass.HAS,
-        from: accountEntity,
-        to: assessmentEntity,
-      });
+    const assessmentEntities: Entity[] = [];
+    const projectEntities: Entity[] = [];
+    const relationships: Relationship[] = [];
```

Comment:
Why do we collect all the `assesmentEntities`, `projectEntities` and `relationships`? Is it better than just `addEntity` and `addRelationship`?